src/components/ladder/RankingsGrid.vue

```diff
@@ -363,26 +363,20 @@ export default class RankingsGrid extends Vue {
     const playersInfo = ranking.playersInfo;
     if (!playersInfo) return "Random";
     const playerInfo = playersInfo[playerIndex];
-    if (
-      RankingsGrid.hasSelectedIcon(playerInfo) &&
-      playerInfo.selectedRace <= ERaceEnum.UNDEAD
-    ) {
+    if (RankingsGrid.hasSelectedIcon(playerInfo)) {
       return this.$t(`races.${ERaceEnum[playerInfo.selectedRace]}`);
     } else {
       return this.$t(`races.${ERaceEnum[playerInfo.calculatedRace]}`);
     }
   }
 
   private static hasSelectedIcon(playerInfo: PlayerInfo) {
-    if (
+    return (
```

Comment:
Isn't it better to extract this method in a helper class and use it in both places? Looks identical to me